package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "firebase": "^4.8.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
     "react-redux": "^5.0.6",
+    "react-redux-firebase": "^1.5.1",
```

Comment:
this just got updated to v2.0 https://www.npmjs.com/package/react-redux-firebase so probably worth updating here and checking no breaking changes first